tests/puppeteer/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/add.js

```diff
@@ -0,0 +1,97 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddSeoUrl extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'SEO & URLs • ';
+
+    // Selectors
+    this.pageTitleLangButton = '#meta_page_title';
+    this.pageTitleLangSpan = 'div.dropdown-menu[aria-labelledby=\'meta_page_title\'] span[data-locale=\'%LANG\']';
+    this.pageNameSelect = '#meta_page_name';
+    this.pageTitleInput = '#meta_page_title_%ID';
+    this.metaDescriptionInput = '#meta_meta_description_%ID';
+    this.metaKeywordsInput = '#meta_meta_keywords_%ID-tokenfield';
+    this.friendlyUrlInput = '#meta_url_rewrite_%ID';
+    // Selectors for Meta keywords
+    this.taggableFieldDiv = 'div.input-group div.js-locale-%LANG';
+    this.deleteKeywordLink = `${this.taggableFieldDiv} a.close`;
+    this.saveButton = 'div.card-footer button';
+  }
+
+  /* Methods */
+
+  /**
+   * Change language for selectors
+   * @param lang
+   * @return {Promise<void>}
+   */
+  async changeLanguageForSelectors(lang = 'en') {
+    await Promise.all([
+      this.page.click(this.pageTitleLangButton),
+      this.page.waitForSelector(`${this.pageTitleLangButton}[aria-expanded='true']`, {visible: true}),
+    ]);
+    await Promise.all([
+      this.page.click(this.pageTitleLangSpan.replace('%LANG', lang)),
+      this.page.waitForSelector(`${this.pageTitleLangButton}[aria-expanded='false']`, {visible: true}),
+    ]);
+  }
+
+  /**
+   * Delete all keywords
+   * @param lang, to specify which input to empty
+   * @return {Promise<void>}
+   */
+  async deleteKeywords(lang = 'en') {
+    const closeButtons = await this.page.$$(this.deleteKeywordLink.replace('%LANG', lang));
+    /* eslint-disable no-restricted-syntax */
```

Comment:
disable this rule or replace it by a foreach maybe ?

Reply:
we want to click on buttons in sequence (in order to click on the right element), we can't do that with foreach (already tested).

Reply:
replaced by 
```js
for (let i = 0; i < closeButtons.length; i++)
```